students/12th/juheejang/style/login.css

```
@@ -0,0 +1,148 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;    
+    background-color: #FAFAFA;
+}
+
+.wrapLogin {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 50px;
+    padding: 10px 0px;
+}
+
+.wrapLoginMain {
+    text-align: center;
+    margin-bottom: 10px;
+    padding: 10px 0px;
+    width: 350px;
+    height: 377.8px;
+    border: 1px solid #DBDBDB;
+    background-color: #FFFFFF;
+}
+
+.logoImage {
+    width: 175px;
+    margin: 22px 86.5px 25px;
+}
```

```suggestion
.logoImage {
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 20px 0;
}
```
margin에 값을 직접적으로 주시는 것 보다는 위와 같이 하시는 걸 추천드립니다!
이 방법 이외에도 logo 이미지에 margin: 0 auto;와 padding을 주는 방법도 있습니다!

---

확인 후 수정하였습니다 😀